port/zos390/omrintrospect.h

```diff
@@ -34,22 +34,25 @@
 #include "omrintrospect_common.h"
 #include "omrutil.h"
 
-#pragma linkage(getthent, OS_UPSTACK)
 #if defined(OMR_ENV_DATA64)
+#pragma linkage(getthent, OS)
 #pragma map(getthent, "BPX4GTH")
-#else
+#else /* defined(OMR_ENV_DATA64) */
+#pragma linkage(getthent, OS_UPSTACK)
 #pragma map(getthent, "BPX1GTH")
-#endif
+#endif /* defined(OMR_ENV_DATA64) */
 
-#pragma linkage(pthread_quiesce, OS_UPSTACK)
 #if defined(OMR_ENV_DATA64)
 #pragma map(pthread_quiesce, "BPX4PTQ")
-#else
+#else /* defined(OMR_ENV_DATA64) */
+#pragma linkage(pthread_quiesce, OS_UPSTACK)
 #pragma map(pthread_quiesce, "BPX1PTQ")
-#endif
+#endif /* defined(OMR_ENV_DATA64) */
 
+#if !defined(OMR_ENV_DATA64)
 #pragma linkage(pthread_quiesce_and_get_np_X, OS_UPSTACK)
 #pragma map(pthread_quiesce_and_get_np_X, "BPX1PQG")
+#endif /* !defined(OMR_ENV_DATA64) */
```

Comment:
Perhaps this should just be removed (I don't see any uses of `pthread_quiesce_and_get_np_X()`).